healthCare-Backend/src/utils.js

```diff
@@ -0,0 +1,16 @@
+const pool = require('../db');
+const queries = require('./queries');
+
+const addPatientFollowUpAppointments = (patient_id, folowUpAppointments) => {
```

Comment:
As discussed, this function should be a part of controller .
For now we do not need utils file.